ansible/roles/netlogon/files/system-auth

```diff
@@ -2,9 +2,9 @@
 
 auth    [success=4 default=ignore] pam_unix.so try_first_pass nullok
 auth    [success=3 default=ignore] pam_policycache.so try_first_pass action=check
-auth    [success=ok default=die] pam_netauth.so try_first_pass
-auth    [success=1 default=ignore] pam_policycache.so try_first_pass action=update
+auth    [success=1 default=die] pam_netauth.so try_first_pass
```

Comment:
Just to help my understanding, the netauth attempt is previous `success=ok`, why did you switch it to `success=1`? (And vice-versa for policycache)

Just to get more than a 4 line diff of explanation on this PR.

Reply:
success=ok says that the chain should succeed if nothing else changes, but pam_policycache doesn't return a success in the update flow, so it instead falls through to pam_deny.so.  Now the order is to check the cache, then check netauth if the cache falls through.  If netauth succeeds, skip the next 1 entries, which moves over pam_deny, from there update the cache ignoring any result it might return and setup the env, permit is there to force the action if the chain makes it this far so that pam_unix and pam_netauth have identical policy flows.

pam_deny is present as a backstop to ensure the chain dies if all three possible authenticators fail.

Reply:
So if pam_unix or `pam_policycache.so try_first_pass action=check` pass, then skip all the way to `pam_env`, yes?

Reply:
That is correct.